### DISCUSSION TITLE
How to set cookie with options `Expires`, `HttpOnly` and `Secure`?

### DISCUSSION THREAD

**Discussion by lz1998** [tokio-rs/axum#2546] [Q&A]
### Summary

How to set cookie with options `Expires`, `HttpOnly` and `Secure`?

### axum version

0.7

**Comment by dayvejones** [comment]
https://docs.rs/axum-extra/latest/axum_extra/extract/cookie/index.html

+ https://docs.rs/cookie/latest/cookie/
